src/couch_replicator_manager.erl

```diff
@@ -410,10 +438,30 @@ rescan(#state{scan_pid = nil} = State) ->
         true ->
             State;
         false ->
+            % Clean up existing change feed pids.
+            % They have to be removed from the list of
+            % pids otherwise resume_scan will never make progress
+            % (it notices an existing pid and assumes there is
+            % a change feed process running, bump rescan flag
+            % and moves on).
```

Comment:
This is only true because we dump the changes pids yeah? We have to do a restart regardless so if we remove the changes filtering above we'd have to remove this one?